.buildkite/hooks/pre-command

```diff
@@ -58,18 +58,6 @@ if [[ "$BUILDKITE_STEP_KEY" == *"integration-tests"* ]]; then
   export API_KEY_TOKEN=$(vault kv get -field apiKey ${CI_ESS_PATH})
   echo ${API_KEY_TOKEN} > ./apiKey
   export TEST_INTEG_AUTH_ESS_APIKEY_FILE=$(realpath ./apiKey)
-
-  # BK analytics
-  echo "--- Prepare BK test analytics token :vault:"
-  BUILDKITE_ANALYTICS_TOKEN=$(vault kv get -field token kv/ci-shared/platform-ingest/buildkite_analytics_token)
-  export BUILDKITE_ANALYTICS_TOKEN
-fi
-
-if [[ "$BUILDKITE_STEP_KEY" == *"aggregate-reports"* ]]; then
-  # BK analytics
-  echo "--- Prepare BK test analytics token :vault:"
-  BUILDKITE_ANALYTICS_TOKEN=$(vault kv get -field token kv/ci-shared/platform-ingest/buildkite_analytics_token)
-  export BUILDKITE_ANALYTICS_TOKEN
```

Comment:
I don't think we need this approach, it's hard to know when it's needed, but using the explicit BK plugin can help us to know when that's needed, as they need each other (`elastic/vault-secrets#v0.1.0` and `test-collector#v1.11.0`